sigma/backends/netwitness/netwitness.py

```diff
@@ -147,15 +148,15 @@ class NetWitnessBackend(TextQueryBackend):
 
     # Null/None expressions
     # Expression for field has null value as format string with {field} placeholder for field name
-    field_null_expression: ClassVar[Optional[str]] = "{field} is null"
+    field_null_expression: ClassVar[Optional[str]] = "{field} !exists || {field} = '-'"
 
     # Field existence condition expressions.
-    field_exists_expression: ClassVar[Optional[str]] = (
-        "exists({field})"  # Expression for field existence as format string with {field} placeholder for field name
-    )
+    # Expression for field existence as format string with {field} placeholder for field name
+    field_exists_expression: ClassVar[Optional[str]] = "{field} exists"
+
     # Expression for field non-existence as format string with {field} placeholder for field name.
     # If not set, field_exists_expression is negated with boolean NOT.
-    field_not_exists_expression: ClassVar[Optional[str]] = "notexists({field})"
+    field_not_exists_expression: ClassVar[Optional[str]] = "{field} !exists"
```

Comment:
Consider adding a const/variable for the negotiation character and dynamically build the strings containing it.

Reply:
The char is only used in this line therefore no need to replace it in multiple strings